### PR TITLE
Added academic value translation

### PR DESCRIPTION
**Pull request by zhongjun2**

### CODE REVIEW

File: chinese/wg-value/focus-areas/academic-value/academic-open-source-project-impact.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+
+### 这个指标为候选基线指标，可以在[问题 #149](https://github.com/chaoss/wg-value/issues/146)进行评审。根据评论周期计划，这个指标将在下一个常规版本发布。
```
Comment (eyehwan) [inline chinese/wg-value/focus-areas/academic-value/academic-open-source-project-impact.md:2]: not needed.

File: chinese/wg-value/focus-areas/academic-value/academic-open-source-project-impact.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+识别哪些项目依赖于您的项目
+识别引用您项目的期刊文章
+
+## 实现
+
+关于如何衡量学术开源项目影响的一些想法：
+
+* 通过在《开源软件杂志》上发表来衡量影响
+* 下游依赖的软件项目的数量
+* 类似于 H 指数的东西（目前不存在）
```
Comment (eyehwan) [inline chinese/wg-value/focus-areas/academic-value/academic-open-source-project-impact.md:27]: H 指数(H-Index)

File: chinese/wg-value/focus-areas/academic-value/academic-open-source-project-impact.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+* 代码行
+* 社区贡献的数量，不是来自研究团队
+* 下游依赖
+
+
+### 过滤条件
+
+### 可视化
+
+### 提供指标的工具
```
Comment (eyehwan) [inline chinese/wg-value/focus-areas/academic-value/academic-open-source-project-impact.md:45]: 提供度量的工具

File: chinese/wg-value/focus-areas/academic-value/academic-open-source-project-impact.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+* 魔典实验室
```
Comment (eyehwan) [inline chinese/wg-value/focus-areas/academic-value/academic-open-source-project-impact.md:47]: GrimoireLab

File: chinese/wg-value/focus-areas/academic-value/academic-open-source-project-impact.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+* 占卜者
```
Comment (eyehwan) [inline chinese/wg-value/focus-areas/academic-value/academic-open-source-project-impact.md:48]: Augur

File: chinese/wg-value/focus-areas/academic-value/academic-open-source-project-impact.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+* [CiteAS](https://citeas.org/)
+* [开源软件杂志 (JOSS)](https://joss.theoj.org/)
+* GitHub 见解
+* 开放科学中心 OSF（开放科学框架）
+* Linux 基金会洞察
+
+
+### 数据收集策略
+
+## 参考
```
Comment (eyehwan) [inline chinese/wg-value/focus-areas/academic-value/academic-open-source-project-impact.md:58]: 参考资料

**Comment by zhongjun2** [comment]
Done

**Comment by eyehwan** [comment]
Looks good to me.

**Pull request merged**